sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3630,8 +3630,16 @@ def _teardown(self,
             # should be higher priority than the cluster requests, and we should
             # release the lock from other requests.
             exclude_request_to_kill = 'sky.down' if terminate else 'sky.stop'
-            requests_lib.kill_cluster_requests(handle.cluster_name,
-                                               exclude_request_to_kill)
+            try:
+                requests_lib.kill_cluster_requests(handle.cluster_name,
```

Comment:
It kind of feels like this should be handled in API server code (like core.down) rather than in the backend. In either case, I definitely think it's better to just avoid calling this path when we are not in an API server (such as from jobs controller or serve controller), rather than catching the exception.

Reply:
Yea, added a TODO there. Let's merge this PR first to unblock the backward compatibility issue.